Remove TemporaryGettingStarted.md. (#706)
swiftlang/swift-testing: CONTRIBUTING.md

### Using Xcode (easiest)

> 1. Install Xcode 16 or newer. You can find Xcode on the [Apple Developer](https://developer.apple.com/xcode/)

Reviewer: ```suggestion
1. Install Xcode 16 or newer, from the [Apple Developer](https://developer.apple.com/xcode/)
```

Not a very strong suggestion, I just think it's a bit more straightforward.

Author: Sure, will tweak.
